packages/app/src/app/pages/Dashboard/types.ts

```diff
@@ -140,6 +141,13 @@ export type DashboardContributionBranch = {
   >['recentBranches'][number];
 };
 
+export type DashboardV2Repository = {
+  type: 'v2-repo';
+  repo: NonNullable<
```

Comment:
I would export a `Repository` type from somewhere so we can reuse based on the graphql generated typing

Reply:
Created a fragment and used it as the type in commit: [14c8fa7](https://github.com/codesandbox/codesandbox-client/pull/6917/commits/14c8fa7fe40935878d890de0e5008a6003b18ec7).